### DISCUSSION TITLE
Is it possible to destroy ad views instead of hide

### DISCUSSION THREAD

**Discussion by yigit-serin** [admob-plus/admob-plus#331] [Ideas]
**Describe the bug**
If you create a lot of ad units, every unit create an ad view (interstitial, banner etc.) this situation uses a lot of ram usage.
There is no destroy option to lower memory usage.

**To Reproduce**
1.   interstitial = new admob.InterstitialAd({
    adUnitId: '123',
  })

  await interstitial.load()
  await interstitial.show()

2. interstitial = new admob.InterstitialAd({
    adUnitId: '987',
  })

  await interstitial.load()
  await interstitial.show()
3. interstitial = new admob.InterstitialAd({
    adUnitId: '456',
  })

  await interstitial.load()
  await interstitial.show()
4. Check created view count and ram usage.

**Expected behavior**
There must be an option to destroy the created ad views.

<img width="347" alt="Screen Shot 2021-05-04 at 14 33 32" src="https://user-images.githubusercontent.com/20225497/116997799-ee463980-ace5-11eb-9e76-b9adde82ba4d.png">

**Comment by ratson** [comment]
No at the moment as I don't expect there will more than 5+ ad units for most of the case.
Ad instance should be reused instead of recreating them every time.
@yigit-serin Is there any special case you need to create many ad units?

**Comment by yigit-serin** [comment]
I used 3 types of ad. (banner, rewarded, interstitial)
To measure ad unit performance, I created some ad units to use on app sections.
Every instance create a new ad view.
Destroy ad views might be a good feature.

**Comment by ratson** [comment]
As this is feature request, move it to `Discussions`.
